docs/authoring-components.md

```diff
@@ -775,31 +775,22 @@ conventions, examples, and common TypeScript patterns you may not be used to, in
 ### Testing
 
 The following guidelines should be used to help write tests for MDC Web code. Our tests are written
-using [mocha](https://mochajs.org/) with the [qunit UI](https://mochajs.org/#qunit), and are driven by [karma](https://karma-runner.github.io/1.0/index.html). We use the [chai assert API](http://chaijs.com/api/assert/)
-for assertions, and [testdouble](https://github.com/testdouble/testdouble.js/) for mocking and stubbing.
+using the [Jasmine](https://jasmine.github.io/) testing framework and are driven by [Karma](https://karma-runner.github.io/1.0/index.html).
```

Comment:
Thanks for updating this doc!

Reply:
The site-generator test was failing without these changes. Otherwise, I wouldn't have noticed 😛